src/electionguard-ui/ElectionGuard.UI/ViewModels/BallotUploadViewModel.cs

```diff
@@ -213,9 +214,10 @@ private async Task Upload()
                 _ = Interlocked.Increment(ref totalCount);
                 UploadText = $"{AppResources.SuccessText} {totalCount} / {ballots.Length} {AppResources.Success2Text}";
             }
-            catch (Exception)
+            catch (Exception ex)
             {
                 _ = Interlocked.Increment(ref totalRejected);
+                _logger.LogWarning(ex, "Ballot being rejected {currentBallot}", currentBallot);

```

Comment:
so sometimes, generic catches are being logged as warnings, other times errors. Should this be a warning?

Reply:
This one since it is being used for the rejected ballots, a warning is enough.  It is not an error.